third_party/nist_cavp_testvectors/repos.bzl

```diff
@@ -5,12 +5,21 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 def nist_cavp_repos():
+    """Load NIST CAVP test vectors
+
+    The NIST website that serves the test vectors (csrc.nist.gov) is sometimes
+    unreliable. To improve availability, the NIST test vectors are hosted in a
+    lowRISC GCS storage bucket (ot-crypto-test-vectors). To add new test
+    vectors, upload the zip file to the GCS bucket, update the manifest there
+    to indicate the original download URL, and use the public link to fetch the
+    archive instead of the NIST website.
+    """
     http_archive(
         name = "nist_cavp_ecdsa_fips_186_4",
         build_file = Label("//third_party/nist_cavp_testvectors:BUILD.nist_cavp_common.bazel"),
         strip_prefix = "186-4ecdsatestvectors",
         sha256 = "fe47cc92b4cee418236125c9ffbcd9bb01c8c34e74a4ba195d954bcb72824752",
-        url = "https://csrc.nist.gov/CSRC/media/Projects/Cryptographic-Algorithm-Validation-Program/documents/dss/186-4ecdsatestvectors.zip",
+        url = "https://storage.googleapis.com/ot-crypto-test-vectors/nist/186-4ecdsatestvectors.zip",
```

Comment:
I'm not sure if it would be useful, but the `urls` argument allows specifying multiple possible sources, which would be tried in order. We could put the official NIST source as a secondary.

Reply:
Oh nice! I did not know about that feature. I'll try the NIST site first and fall back to gcloud as a backup to reduce some gcloud costs on our end.